concourse/scripts/compile_gpdb.bash

```diff
@@ -131,7 +131,6 @@ function export_gpdb_clients() {
 }
 
 function _main() {
-
```

Comment:
Is this change necessary to fix compiler warnings? I think it should be done in other patch